server/app/routes/v1/ping_api.rb

```diff
@@ -1,13 +1,20 @@
 module V1
   class PingApi < Roda
+    include RequestHelpers
 
     plugin :json
     plugin :render, engine: 'json.jbuilder', views: 'app/views/v1/ping'
 
     route do |r|
       r.is do
         r.get do
-          render('show')
+          begin
+            Grid.count # test db connection
+            MongoPubsub.started? # test pubsub
```

Comment:
[`MongoPubsub.started?`](https://github.com/kontena/kontena/blob/dd68aadceb3f8d0cc6e447b54095ae60ea2f1e37/server/app/services/mongo_pubsub.rb#L137) doesn't actually check that the `MongoPubsub` actor is running and is still alive: it just checks that the default celluloid supervision container has been created during `Celluloid.boot`... this is not a good health-check, it's only meant for skipping stuff that happens early during server bootup before `Celluloid.boot`.

Reply:
Good point, fixed.